xmtp_mls/src/groups/mod.rs

```diff
@@ -719,46 +725,35 @@ impl<ScopedClient: ScopedGroupClient> MlsGroup<ScopedClient> {
                 }
             });
 
+            let mut group = StoredGroup::builder();
+            group.id(group_id)
```

Comment:
the builder allows us to separate context-specific arguments into each branch, while not repeating the majority of common arguments